doc/specs/stdlib_sorting.md

```diff
@@ -196,6 +198,18 @@ magnitude slower than `ORD_SORT`. Its memory requirements are also
 low, being of order O(Ln(N)), while the memory requirements of
 `ORD_SORT` and `SORT_INDEX` are of order O(N).
 
+#### The `RADIX_SORT` subroutine
+
+`RADIX_SORT` is a implementation of LSD [radix sort](https://en.wikipedia.org/wiki/Radix_sort),
```

Comment:
Do you have another reference than wikipedia (as the algorithm is only briefly described there)?

Reply:
> Do you have another reference than wikipedia (as the algorithm is only briefly described there)?

Actually I refers to lots of open materials. The following I think best describes the LSD radix sort algorithm:
```suggestion
`RADIX_SORT` is a implementation of LSD [radix sort](https://www.growingwiththeweb.com/sorting/radix-sort-lsd/),
```